Applications/Tizen_native/CustomShortcut/src/data.c

```diff
@@ -43,6 +43,186 @@ int parse_route(const char *source, char **route, char **data) {
   return APP_ERROR_NONE;
 }
 
+void data_get_resource_path(const char *file, char *full_path, bool shared) {
+  char *root_path;
+  if (shared) {
+    root_path = app_get_shared_resource_path();
+  } else {
+    root_path = app_get_resource_path();
+  }
+
+  if (root_path == NULL) {
+    LOG_E("failed to get resource path");
+    return;
+  }
+
+  if (full_path == NULL) {
+    LOG_E("full_path is null");
```

Comment:
When full_path is null, segmentation fault is expected in this logic.
How about just returning error value?

Reply:
Fixed. Thanks!